keyboards/skyloong/gk61/info.json

```diff
@@ -0,0 +1,305 @@
+            {"matrix": [1, 5], "x": 82, "y": 16, "flags": 4},
+            {"matrix": [1, 6], "x": 97, "y": 16, "flags": 4},
+            {"matrix": [1, 7], "x": 112, "y": 16, "flags": 4},
+            {"matrix": [1, 8], "x": 127, "y": 16, "flags": 4},
+            {"matrix": [1, 9], "x": 142, "y": 16, "flags": 4},
+            {"matrix": [1, 10], "x": 156, "y": 16, "flags": 4},
+            {"matrix": [1, 11], "x": 171, "y": 16, "flags": 4},
+            {"matrix": [1, 12], "x": 186, "y": 16, "flags": 4},
+            {"matrix": [1, 13], "x":216, "y": 16, "flags": 4},
+            {"matrix": [2, 0], "x": 0, "y": 32, "flags": 4},
+            {"matrix": [2, 1], "x": 26, "y": 32, "flags": 4},
+            {"matrix": [2, 2], "x": 41, "y": 32, "flags": 4},
+            {"matrix": [2, 3], "x": 56, "y": 32, "flags": 4},
+            {"matrix": [2, 4], "x": 71, "y": 32, "flags": 4},
+            {"matrix": [2, 5], "x": 86, "y": 32, "flags": 4},
+            {"matrix": [2, 6], "x": 101, "y": 32, "flags": 4},
+            {"matrix": [2, 7], "x": 115, "y": 32, "flags": 4},
+            {"matrix": [2, 8], "x": 130, "y": 32, "flags": 4},
+            {"matrix": [2, 9], "x": 145, "y": 32, "flags": 4},
+            {"matrix": [2, 10], "x": 160, "y": 32, "flags": 4},
+            {"matrix": [2, 11], "x": 175, "y": 32, "flags": 4},
+            {"matrix": [2, 13], "x": 208, "y": 32, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 48, "flags": 4},
+            {"matrix": [3, 1], "x": 34, "y": 48, "flags": 4},
+            {"matrix": [3, 2], "x": 48, "y": 48, "flags": 4},
+            {"matrix": [3, 3], "x": 63, "y": 48, "flags": 4},
+            {"matrix": [3, 4], "x": 78, "y": 48, "flags": 4},
+            {"matrix": [3, 6], "x": 93, "y": 48, "flags": 4},
+            {"matrix": [4, 6], "x": 108, "y": 48, "flags": 4},
+            {"matrix": [3, 7], "x": 123, "y": 48, "flags": 4},
+            {"matrix": [3, 8], "x": 138, "y": 48, "flags": 4},
+            {"matrix": [3, 9], "x": 153, "y": 48, "flags": 4},
+            {"matrix": [3, 10], "x": 168, "y": 48, "flags": 4},
+            {"matrix": [3, 12], "x": 205, "y": 48, "flags": 4},
+            {"matrix": [4, 0], "x": 0, "y": 64, "flags": 4},
+            {"matrix": [4, 1], "x": 19, "y": 64, "flags": 4},
+            {"matrix": [4, 2], "x": 37, "y": 64, "flags": 4},
+            {"matrix": [4, 4], "x": 56, "y": 64, "flags": 4},
+            {"matrix": [4, 5], "x":95, "y": 64, "flags": 4},
+            {"matrix": [3, 5], "x": 107, "y": 64, "flags": 4},
+            {"matrix": [4, 8], "x": 130, "y": 64, "flags": 4},
+            {"matrix": [4, 9], "x": 149, "y": 64, "flags": 4},
+            {"matrix": [4, 10], "x": 175, "y": 64, "flags": 4},
+            {"matrix": [4, 11], "x": 198, "y": 64, "flags": 4},
+            {"matrix": [4, 13], "x": 220, "y": 64, "flags": 4}
+        ]
+    },
+    "url": "https://github.com/JZ-Skyloong",
+    "usb": {
+        "vid": "0x1EA7"
```

Comment:
According to https://usb-ids.gowdy.us/read/UD/1ea7, the USB vendor ID `0x1EA7` is already in use.

Reply:
> According to https://usb-ids.gowdy.us/read/UD/1ea7, the USB vendor ID `0x1EA7` is already in use.

Semitek Company in Hong Kong is our parent company.